spinalcordtoolbox/registration/core.py

```diff
@@ -161,9 +161,12 @@ def register_wrapper(fname_src, fname_dest, param, paramregmulti, fname_src_seg=
             src = ['src_label.nii']
             dest = ['dest_label_RPI.nii']
             interp_step = ['nn']
+        elif step.type == 'rootlet':
+            src = ['src.nii', 'src_label.nii']
+            dest = ['dest_RPI.nii', 'dest_label_RPI.nii']
+            interp_step = ['spline']  # Maybe do or linear?
```

Comment:
What are the pros and cons of `spline` vs. `linear`? o:

Reply:
I think usually for images we use spline interpolation, which I would not do for segmentations or labels